frontend/views/containers/user-settings/AppLogs.vue

```diff
@@ -2,6 +2,8 @@
   .settings-container
     section.card
       .c-header
+        p.c-instructions(v-if='ephemeral.reportError')
+          | [Give some instructions to the user about this page. Ask @mmbotelho for help]
```

Comment:
What you had before here was really good. Please add it back, or some version of it, showing the error message that was passed in via the query parameters.

E.g. `Search the logs below for "${errorMsg}" for details. Please send us the logs so that we can help you debug any problems.`

Reply:
Let’s wait for @mmbotelho feedback on this one :)

Reply:
Can you give me a bit more context on what this is? @sandrina-p

Reply:
Oh, this is the Application page. As I said on the issue #788, I think it's missing some instructions there so the user know what to do. We thought about something:

> You faced a recent error "${errorMsg}". Please send us the logs at [link], so that we can help you solve the problem.

You can pull this PR and see it by yourself. Try to change the group mincome, it will appear a an error.

Reply:
Ok that's clear now! Thank you for explaining. What about:

"Failed to change the mincome. You can report the error **here**."

As for the actual app log page, I think we need to add some instructions there, as you already have. Are you ok with not doing this on this PR though, and removing the message you added to the app logs page for now? We can create an issue for it, I can work on it and then pass it on to you. This is a bit of a chaotic way of working - I feel like the UX is being decided on code reviews and I don't think it's the most productive way of doing things. 

What do you think?

Reply:
@mmbotelho I think we are in "crunch time" and that means two things:

1. Yes, you are right, it is a chaotic way of doing things.
2. We should make minor changes like this anyway, but document them in issues to have them adjusted and fixed in future PRs.
3. The problem is that right now we need you @mmbotelho to focus on the UI design tasks you're focusing on, and *not* jump around to little things like this. When you get around to this (whenever that happens), then you can focus exclusively on this. But we need to move forward with this now and not be blocked on it. We want everyone to do a good job. The next few months are going to be a little bit more chaotic, but we will try to keep the chaos down to a minimum by keeping track of all such deviations via GH issues. This is why it's really important for @sandrina-p and me and all the other developers to open issues and assign them to you to review, if the changes are *both* important and minor.

I really like this text:

> You faced a recent error: "{errorMsg}". Please download the logs and {a_}send to us{_a}, so that we can help you solve the problem.

I would only change the first part to:

> Error: "{errorMsg}". Please download the logs and {a_}send to us{_a}, so that we can help you solve the problem.

@sandrina-p Please go ahead and implement this, but make sure to open an issue and assign to @mmbotelho to adjust UI for this as she sees fit.

Reply:
> The problem is that right now we need you @mmbotelho to focus on the UI design tasks you're focusing on, and not jump around to little things like this. When you get around to this (whenever that happens), then you can focus exclusively on this. But we need to move forward with this now and not be blocked on it. We want everyone to do a good job. The next few months are going to be a little bit more chaotic, but we will try to keep the chaos down to a minimum by keeping track of all such deviations via GH issues. This is why it's really important for @sandrina-p and me and all the other developers to open issues and assign them to you to review, if the changes are both important and minor.

Ok. Can we leave the adjustments to the App logs page out of it then? I think that's out of the scope of the original issue anyway

Reply:
@mmbotelho 

> Ok. Can we leave the adjustments to the App logs page out of it then? I think that's out of the scope of the original issue anyway

Per our Slack chat, is it OK with you if we keep these changes in this PR and document them?

These changes are part of closing #788, as documented there:

> The debug page can use the passed in `error` query parameter to generate a draft of a message that the user can submit to us

We just forgot the design part of it, that's what was the mistake I think. Is it OK if in this case we move forward with it and fix it up in a future PR?

Reply:
Sure! Can you please create an issue for it @sandrina-p ? Feel free to assign it to me

Reply:
So, let me make a recap:
- The instructions I came up with will be implemented in this PR as asked by @taoeffect.
- I created an issue #890 about this change, so @mmbotelho can take her time to properly review these instructions and any copy related to the Applications pages or scoped error messages.
- We, as a team, will make an effort to avoid chaotic workflows in future tasks. When picking a new task/issue, let's make sure all the UI/UX related to it is as clear as possible before moving to development phase.

Reply:
That sounds great to me @sandrina-p 😄  👍